sentinel/README.md

```diff
@@ -0,0 +1,313 @@
+# Sentinel
+
+Sentinel is a robust, modular, and extensible Go-based framework designed for monitoring blockchain events and managing event subscriptions. It orchestrates multiple `ChainPollerService` instances, each tied to a specific blockchain network, to fetch logs and notify relevant subscribers seamlessly.
+
+## Table of Contents
```

Comment:
I really want to test it with testnets before we merge, at least `Fuji`, `Sepolia`, some `Optimism` and `Arbitrum`